src/modules/common/components/filterDropdownButton/selectFilter.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { withTranslation } from 'react-i18next';
-import { MODULE_COMMANDS_NAME_ID_MAP } from '@transaction/configuration/moduleAssets';
-import { getModuleCommandTitle } from '@transaction/utils/moduleAssets';
+import { MODULE_COMMANDS_NAME_MAP } from 'src/modules/transaction/configuration/moduleCommand';
+import { getModuleCommandTitle } from 'src/modules/transaction/utils/moduleCommand';
 import Select from 'src/theme/Select';
 import styles from './filters.css';
 
 const SelectFilter = ({
   label, placeholder, filters, name, updateCustomFilters,
 }) => {
-  const options = Object.keys(MODULE_COMMANDS_NAME_ID_MAP)
+  const options = Object.keys(MODULE_COMMANDS_NAME_MAP)
```

Comment:
not required to change, but `Object.entries` makes the trick here ;)

Reply:
As i mentioned below this logic should be refactored, so i am not going to change anything for now, added a TODO.

Reply:
🆗